ksqldb-engine/src/main/java/io/confluent/ksql/query/KsqlSerializationClassifier.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2022 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.query;
+
+import io.confluent.ksql.query.QueryError.Type;
+import io.confluent.ksql.serde.KsqlSerializationException;
+import io.confluent.ksql.util.ReservedInternalTopics;
+import java.util.Objects;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.apache.kafka.streams.errors.StreamsException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * {@code SerializationClassifier} classifies serialization exceptions as user error
```

Comment:
leftover from past commit: change {@code SerializationClassifier} to {@code KsqlSerializationClassifier}

Reply:
Done